### PR TITLE
lxc/migrate: Make live migration error message more helpful

### PR DESCRIPTION
**Pull request by yunginnanet**
Signed-off-by: kayos@tcp.direct <kayos@tcp.direct>

### CODE REVIEW

**Comment by tomponline** [comment]
jenkins: test this please

**Comment by yunginnanet** [comment]
I can't decipher where this is failing

File: lxd/migration/utils.go [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package migration
 
+import "fmt"
```
Comment (tomponline) [inline lxd/migration/utils.go:3]: Please can you use

```
import (
	"fmt"
)
```

File: lxd/migration/utils.go [hunk 2]
```diff
@@ -86,3 +88,17 @@ func (m *MigrationHeader) GetBtrfsFeaturesSlice() []string {
...
 }
+
+const (
+	unableToLiveMigrate = "Unable to perform live container migration."
+	toMigrateLive       = "To migrate the container, stop the container before migration or install CRIU"
+)
+
+var (
+	ErrNoLiveMigrationSource = fmt.Errorf("%s CRIU isn't installed on the source server. %s on the source server",
+		unableToLiveMigrate, toMigrateLive)
```
Comment (tomponline) [inline lxd/migration/utils.go:99]: Please can you move these onto single lines. Thanks

File: po/lxd.pot [hunk 3]
```diff
@@ -7,7 +7,7 @@
 msgid   ""
 msgstr  "Project-Id-Version: lxd\n"
         "Report-Msgid-Bugs-To: lxc-devel@lists.linuxcontainers.org\n"
-        "POT-Creation-Date: 2023-01-16 14:20+0000\n"
+        "POT-Creation-Date: 2023-01-17 02:32-0800\n"
```
Comment (tomponline) [inline po/lxd.pot:10]: No need for this change as far as I can see.

Comment (yunginnanet) [reply po/lxd.pot:10]: copy that, I didn't think so either but I was trying to figure out the static analysis fail in CI.

**Comment by tomponline** [comment]
jenkins: test this please

**Review by tomponline — CHANGES_REQUESTED** [review]
Please can you remove the i18n commit and squash the other 2 into one. Thanks

**Comment by tomponline** [comment]
jenkins: test this please

**Pull request merged**